lib/handbook/addon/docs/testing/template.md

```diff
@@ -1,3 +1,32 @@
 # Testing
 
-TODO
+## Test categories
+
+### By what they try to determine
+We organize the tests in the hierarchy by this set of categories.
+
+#### Acceptance test
+These are a type of Application test (see below) that looks at the behavior of the whole app from a user's perspective. Ideally, when adding a new page, the Acceptance tests should be the first thing you write after you get all of the appropraite information from Product and preferably after consulting with QA. The handy thing about acceptance tests is that it's behavior driven, so it shouldn't matter what the underlying implementation of the models or the components are. If the OSF API is defined, you should be able to define what the test selectors are and how they react to various situations, then fill in the functionality of everything else to make those tests pass.
+
+If your Acceptance test isn't robust enough to survive a refactor of the ember-osf-web application (to typescript, to a new version of ember, to engines, or whatever), then it's probably not been designed right. Good tests here will eliminate a lot of testing that QA would have to otherwise do.
```

Comment:
> Good tests here will eliminate a lot of testing that QA would have to otherwise do.

Or, more accurately, will prevent additional rounds of development and QA testing. (Sounds like you meant acceptance tests kinda replace QA, which I don't think is true)

Reply:
No, I think it will eliminate the need for some testing work if it's done well and is reliable. That testing work may just be creation of selenium tests, but it might also be a lot of regression testing.